system/Test/FeatureResponse.php

```diff
@@ -68,7 +69,7 @@ class FeatureResponse extends TestCase
 	 *
 	 * @param Response $response
```

Comment:
remove unneded @param as already type hinted, and the type hint already changed

Reply:
`@var` in response property need to be updated

Reply:
update complete